### PR TITLE
Prepared a readme file

### PR DESCRIPTION
**Pull request by ivan0dyatlyukkk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,114 @@
...
+   ```
+Great! We already set our environment up! ⭐
+ 
+## ⚙Integration with Postman
+   If you don't want to come up with an idea of data for the testing requests, and you don't want to type lots of similar requests again and again, you can use the prepared [collection for Postman](#postman-collection) and the [environment](#postman-environment)
+
+## 🎯Summary
+<strong><i>The Online Bookstore API</i></strong> is a powerful tool designed to create a seamless and user-friendly experience for book enthusiasts and administrators alike. It serves as the backbone for managing a virtual library of literary treasures and facilitates the process of discovering, purchasing, and managing books. 
+
+The video instructions for usage you can watch here: link-for-the-loom-video
```
Comment (andrii-hoienko) [inline README.md]: Where is the link? Or am I missing something?

File: README.md [hunk 2]
```diff
@@ -1 +1,114 @@
...
+    docker-compose build
+
+5. Run the application: <br>
+   ```bash
+   docker-compose up
+   ```
+Great! We already set our environment up! ⭐
+ 
+## ⚙Integration with Postman
+   If you don't want to come up with an idea of data for the testing requests, and you don't want to type lots of similar requests again and again, you can use the prepared [collection for Postman](#postman-collection) and the [environment](#postman-environment)
```
Comment (andrii-hoienko) [inline README.md:107]: Add something about Contributing

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
GJ! Let`s improve your solution

File: README.md (hunk 1 above)
Comment (ivan0dyatlyukkk) [reply README.md]: Sorry, I haven't prepared a loom video yet, and I created the readme for checking my main concept

**Review by vovasalyha — APPROVED** [review]

**Pull request merged**